### PR TITLE
Script to run adapter configuration for Avago_9361 adapter V2

### PR DESCRIPTION
**Pull request by vrbagalkote**
Adjustable values such as cc, bgi, pr etc are set

Signed-off-by: Venkat R B <vrbagal1@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by vrbagalkote** [comment]
Attached is the job.log file.

[job.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/636767/job.txt)

**Comment by narasimhan-v** [comment]
Looks good to me

File: io/disk/Avago_storage_adapter/avago9361.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        Function to set all adjustable values of the adapter
+        """
+        adjustable_rate = ['rebuildrate', 'migraterate', 'ccrate',
+                           'bgirate', 'prrate']
+        for i in adjustable_rate:
+            for j in [0, 10, 30, 60, 100]:
+                cmd = "./storcli64 /c%d show %s" % (self.controller, i)
+                self.check_pass(cmd, "Failed to show the rate")
+                cmd = "./storcli64 /c%d set %s=%d" % (self.controller, i, j)
+                self.check_pass(cmd, "Failed to set the rate")
```
Comment (PraveenPenguin) [inline io/disk/Avago_storage_adapter/avago9361.py]: one help is both command is interrelated like if ` ./storcli64 /c<controller> show rebuildrate` fail than  `./storcli64 /c<controller> set rebuildrate=0" ` not make sense ? just curious about this command

Comment (vrbagalkote) [inline io/disk/Avago_storage_adapter/avago9361.py]: Both commands are different. Even if show command fails for some reason, set will work independently.

Comment (PraveenPenguin) [inline io/disk/Avago_storage_adapter/avago9361.py]: @vrbagalkote sure thanks

**Comment by PraveenPenguin** [comment]
@narasimhan-v thanks

**Pull request merged**